files/ceph-version-qemu.sh

```diff
@@ -3,38 +3,82 @@
 # By default, search for upgrades from Jewel to Luminous.
 CEPH_MAJORVER_EXPECTED_OLD=10
 CEPH_MAJORVER_EXPECTED_NEW=12
+# By default, exclude VMs that do not have RBD block devices attached.
```

Comment:
This does not reflect my test. When I use -e it returns two instances that does not have a rbd volume, while when not using -e it returns nothing even if one instances has an rbd volume.

Reply:
Too much caffeine when writing, the logic was backwards. Should be fixed now.